wherehows-frontend/app/controllers/api/v2/Dataset.java

```diff
@@ -246,6 +250,55 @@ private static AclDao initAclDao() {
     return Promise.promise(() -> ok(Json.newObject().set("health", Json.toJson(health))));
   }
 
+  public static Promise<Result> getExportPolicy(String datasetUrn) {
+    final DatasetExportPolicy exportPolicy;
+    try {
+      exportPolicy = EXPORT_POLICY_DAO.getDatasetExportPolicy(datasetUrn);
+    } catch (Exception e) {
+      if (e.toString().contains("Response status 404")) {
+        return Promise.promise(() -> notFound(_EMPTY_RESPONSE));
+      }
+
+      Logger.error("Fetch export policy fail", e);
+      return Promise.promise(() -> internalServerError(errorResponse(e)));
+    }
+
+    if (exportPolicy == null) {
+      return Promise.promise(() -> notFound(_EMPTY_RESPONSE));
+    }
+
+    return Promise.promise(() -> ok(Json.newObject().set("exportPolicy", Json.toJson(exportPolicy))));
+  }
+
+  public static Promise<Result> updateExportPolicy(String datasetUrn) {
+    final String username = session("user");
+
+    if (StringUtils.isBlank(username)) {
+      return Promise.promise(() -> unauthorized(_EMPTY_RESPONSE));
+    }
+
+    try {
+      JsonNode record = request().body().asJson();
+
+      boolean containsUserGeneratedContent = record.get("containsUserGeneratedContent").asBoolean();
+      boolean containsUserActionGeneratedContent = record.get("containsUserActionGeneratedContent").asBoolean();
+      boolean containsUserDerivedContent = record.get("containsUserDerivedContent").asBoolean();
+
+      final DatasetExportPolicy exportPolicy = new DatasetExportPolicy();
+
+      exportPolicy.setContainsUserGeneratedContent(containsUserGeneratedContent);
+      exportPolicy.setContainsUserActionGeneratedContent(containsUserActionGeneratedContent);
+      exportPolicy.setContainsUserDerivedContent(containsUserDerivedContent);
+
+      EXPORT_POLICY_DAO.updateDatasetExportPolicy(exportPolicy, username);
+    } catch (Exception e) {
+      Logger.error("Update dataset export policy fail", e);
+      return Promise.promise(() -> internalServerError(errorResponse(e)));
+    }
+
+    return Promise.promise(() -> ok(_EMPTY_RESPONSE));
```

Comment:
Would it be useful to return the newly persisted value as the endpoint response, that way consumers do not need to make a separate GET request to sync up with the server. Especially once we start integrating with redux